### DISCUSSION TITLE
a way for Wikitraccs to remove the Trial banner on migrated pages?

### DISCUSSION THREAD

**Discussion by jlriverayuso** [WikiTransformationProject/wikitraccs-releases#139] [Q&A]
Hi,

It appears that I started a migration without the license and now all of the pages contain the Trial banner. Would I need to delete all the pages and migrate them again? or would Wikitraccs remove it when it sees that I added the license if I redo the Transformation?

**Comment by heinrich-ulbricht** [comment]
@jlriverayuso I fear that there is no way to remove the trial markers. WikiTraccs not only adds the trial banner but also replaces random words on the pages. The trial mode is designed to be impossible to remove, to prevent working around it. Sorry :(
